Lab1ForReview/Lab1b/Syntax.in.txt

```diff
@@ -0,0 +1,25 @@
+<program> ::= <decllist> "|" <cmpdstmt> "|"
+<decllist> ::= <declaration | <declaration> "|" <decllist>
+<declaration> ::= IDENTIFIER "space" <type> 
+<type1> ::= "nr" | "NR" |"BOOLEAN"| "string" 
+<arraydecl> ::= <type1> "{" nr "}" //numai recursiv
+<type> ::= <type1> | <arraydecl> 
+<cmpdstmt> ::= <stmtlist> 
+
```

Comment:
Like in lexic, there are some language elements written in BNF ( ::= ) and some in EBNF (“ ”)